test-ciphers.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
atm not calling this file.

Reply:
@spadgett think we need to get the test running?  Since we are missing the `test-prow-e2e.sh` file at 3.11, `test-ciphers.sh` is not running.